test/sim/items/shellbell.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Shell Bell', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it.skip(`should heal from the damage against all targets of the move`, function () {
```

Comment:
But just the last hit of a multi-hit move? What about Parental Bond?

Reply:
Actually, Parental Bond and Shell Bell should also consider all of their hits. I'll add tests for those too! I imagine that means Shell Bell behavior changed at some point after Gen 7 (I tested in 7 just to make sure I had an accurate Parental Bond).

Reply:
Here's a video of Shell Bell + Rock Blast to clarify a bit: https://www.youtube.com/watch?v=_VT6QtL1KJo